pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.cs

```diff
@@ -426,7 +426,7 @@ private void LoadLibrary()
                     {
                         var message = TextUtil.LineSeparate(string.Format(Resources.ViewLibraryDlg_LoadLibrary_An_error_occurred_attempting_to_import_the__0__library, selectedLibrarySpec.Name),
                                         x.Message);
-                        MessageDlg.Show(this, message);
+                        MessageDlg.ShowWithException(this, message, x);

```

Comment:
Will this show the exception message twice, as it's already part of the value of message?

Reply:
> Will this show the exception message twice, as it's already part of the value of message?

When the MessageDlg is displayed, the only thing that the user will see is "message".
The exception would only be shown if the user pressed the "More Info" button.